hypothesis/via
# Add a list of URLs and a way for scanning them for validity

This allows us to:

 * Know which URLs are publicly accessible
 * Return HTML
 * And to cache the contents etc.

The list contains 1024 more popular URLs taken from: https://metabase.hypothes.is/question/861 in a CSV (which is why the diff is so big).

### To test locally

 * `pip install diskcache`
 * Run "bin/scan.py"

This should scan each of the URLs listed in the CSV and build a local disk cache of the results. You can stop at any time and it will resume from where it left off. 

The final output is an updated CSV file which has excluded any URLs which can't be reached (this is the result in the repo right now).
